api-specificatie/openapi.yaml

```diff
@@ -486,51 +483,38 @@ components:
           $ref: "#/components/schemas/Nummeraanduiding"
         woonplaatsen:
           $ref: "#/components/schemas/Woonplaats"
+        adresseerbaarobjecten:
+          oneOf:
+            - $ref: "#/components/schemas/Ligplaats"
+            - $ref: "#/components/schemas/Verblijfsobject"
+            - $ref: "#/components/schemas/Standplaats"
+          discriminator: 
+            propertyName: type
     #JBO: Zoeksuggestie wordt hier als een redelijk generiek antwoord op een generieke zoekfuntie gebruikt. Is het te bedoelding dit te hergerbuiken in andere zoek-functie. 
     #JBO: Wat hier "id" genoemd wordt het in de raadpleeg-functie de "key". Is dit verschil een gevolg van de keuze om dit generiek toe te kunnen passen ? 
-    ZoekSuggestie:
+    Keuzelijst:
       type: object
       properties:
-        type:
-          description: 'Type van gevonden zoeksuggestie'
-          title: 'type'
-          type: string
         weergavenaam:
           description: 'Weergavenaam van de zoeksuggestie'
           title: 'weergavenaam'
           type: string
-        id:
-          description: 'Id van de zoeksuggestie'
-          title: 'id'
+        nummeraanduidingidentificatie:
+          description: 'Identificatie van de nummeraanduiding die aan dit adres gerelateerd is. '
+          title: 'nummeraanduidingidentificatie'
           type: string
-        score:
-          description: 'Score van de zoeksuggestie. Hoger is dichterbij de zoekterm'
-          title: 'score'
-          type: number
-
-    ZoekSuggestieResponse:
-      description: 'Lijst van suggestie op basis van een zoekstring'
-      title: ZoekSuggestieResponse
+        adresseerbaarobjectidentificatie:
+          description: 'Identificatie van het adresseerbaarobject dat aan dit adres gerelateerd is.'
+          title: 'adresseerbaarobjectidentificatie'
+          type: string
+    KeuzelijstCollectieHal:
+      description: 'resulaten als lijst'
       type: object
-      properties:
-        resultCount:
-          description: 'aantal gevonden resultaten'
-          title: resultCount
-          type: integer
-        resultStart:
-          description: 'nummer van het eerste resultaat in de lijst'
-          title: resultStart
-          type: integer
-        maxScore:
-          description: 'hoogste score van de suggesties'
-          title: maxScore
-          type: number
-        idStringPairs:
-          description: 'resulaten als lijst'
-          title: idStringPairs
+      properties: 
+        Keuzelijst:
           type: array
           items:
-            $ref: '#/components/schemas/ZoekSuggestie'
+            $ref: '#/components/schemas/Keuzelijst'
```

Comment:
De ref zou naar een keuzeItem moeten zijn en niet naar een lijst.

Reply:
Goed voorstel. Overgenomen. Als ik mijn wijzigingen commit is dit te zien.